util/src/main/java/io/grpc/util/MultiChildLoadBalancer.java

```diff
@@ -193,7 +193,7 @@ private List<ChildLbState> updateChildrenWithResolvedAddresses(
       newChildLbStates.add(childLbState);
       if (entry.getValue() != null) {
         childLbState.setResolvedAddresses(entry.getValue()); // update child
-        childLbState.lb.handleResolvedAddresses(entry.getValue()); // update child LB
+        childLbState.lb.acceptResolvedAddresses(entry.getValue()); // update child LB
```

Comment:
This class needs to fix its handling of the `Status`. I think we should leave it as-is and create a tracking bug for it.

Reply:
Leaving it as-is(handleResolvedAddresses) is resulting in build failure because of the deprecated warning, so i'm retaining this acceptResolvedAddresses for now to proceed further. Please confirm if this is ok and handling the ```Status``` can be done as part of a new defect.

Reply:
Hmmm, we can definitely suppress warning here and create a tracking bug. Add a comment with TODO mentioning the tracking bug.